cmake/Modules/CMakeCargo.cmake

```diff
@@ -22,31 +38,37 @@ function(cargo_build)
         set(LIB_BUILD_TYPE "release")
     else()
         set(LIB_BUILD_TYPE "debug")
-    endif()    
-    
+    endif()
+
     if(WIN32)
-        set(LIB_FILE "target/${LIB_TARGET}/${LIB_BUILD_TYPE}/${LIB_NAME}.lib")
+        set(LIB_FILE "${CARGO_TARGET_DIR}/${LIB_TARGET}/${LIB_BUILD_TYPE}/${LIB_NAME}.lib")
     else()
-        set(LIB_FILE "target/${LIB_TARGET}/${LIB_BUILD_TYPE}/lib${LIB_NAME}.a")
+        set(LIB_FILE "${CARGO_TARGET_DIR}/${LIB_TARGET}/${LIB_BUILD_TYPE}/lib${LIB_NAME}.a")
```

Comment:
CMAKE_SHARED_LIBRARY_PREFIX could be used instead of hardcoding "lib". On Windows, this variable is empty.